### ISSUE TITLE
[compat] Tables = "0.2" does not seem to accept 1.0.0

### ISSUE THREAD

**Issue by mmikhasenko** [JuliaHEP/UpROOT.jl#3]
```julia
(v1.3) pkg> develop --local UpROOT
  Updating registry at `C:\Users\mikha\.julia\registries\General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
  Updating git-repo `https://github.com/JuliaHEP/UpROOT.jl.git`
 Resolving package versions...
ERROR: Unsatisfiable requirements detected for package Tables [bd369af6]:
 Tables [bd369af6] log:
 ├─possible versions are: [0.1.0-0.1.15, 0.1.17-0.1.19, 0.2.0-0.2.11, 1.0.0-1.0.2] or uninstalled
 ├─restricted to versions 0.2 by UpROOT [69ce31c6], leaving only versions 0.2.0-0.2.11
 │ └─UpROOT [69ce31c6] log:
 │   ├─possible versions are: 0.2.1 or uninstalled
 │   └─UpROOT [69ce31c6] is fixed to version 0.2.1
 └─restricted by compatibility requirements with CSV [336ed68f] to versions: 1.0.0-1.0.2 — no versions left
   └─CSV [336ed68f] log:
     ├─possible versions are: [0.3.0-0.3.1, 0.4.0-0.4.3, 0.5.0-0.5.24] or uninstalled
     └─restricted to versions 0.5.24 by an explicit requirement, leaving only versions 0.5.24
```

**Comment by oschulz** [comment]
Thanks, will update!

**Comment by mmikhasenko** [comment]
Ok, thanks.
```
Tables = "0.2, 1.0"
```
 solves the problem.

**Comment by oschulz** [comment]
Yep, registering a new version right now:  JuliaRegistries/General#10060

**Comment by oschulz** [comment]
Fixed in UpROOT v0.2.2
